assets/_js/components/browser.js

```diff
@@ -13,15 +13,17 @@ OLCS.browser = (function(document, $, undefined) {
 
   var exports = {};
 
-  exports.isOpera = !!window.opera || navigator.userAgent.indexOf(" OPR/") >= 0;
+  exports.isOpera = (!!window.opera || navigator.userAgent.indexOf(" OPR/") >= 0);
 
-  exports.isFirefox = typeof InstallTrigger !== "undefined";
+  exports.isFirefox = (navigator.userAgent.toLowerCase().indexOf("firefox") > -1);
 
-  exports.isSafari = Object.prototype.toString.call(window.HTMLElement).indexOf("Constructor") > 0;
+  exports.isSafari = (/constructor/i.test(window.HTMLElement) ||
+      (function (p) { return p.toString() === "[object SafariRemoteNotification]"; })
+      (typeof window.safari !== "undefined" && typeof window.safari.pushNotification !== "undefined"));
 
-  exports.isChrome = !!window.chrome && !exports.isOpera;
+  exports.isChrome = (!!window.chrome && !exports.isOpera && /Google Inc/.test(navigator.vendor));
 
-  exports.isIE = /*@cc_on!@*/false || !!document.documentMode;
+  exports.isIE = (/*@cc_on!@*/false || !!document.documentMode);
```

Comment:
Was this broken? This isn't related to jQuery is it?

Reply:
Deprecation warnings were present in the console regarding some of the checks.